ic/4510_BCD_Counter/4510_BCD_Counter.xml

```diff
@@ -64,20 +64,20 @@
 			<line start="_Middle+50x+60y" end="_End+60y" />
 
 			<!-- Draw a label at each pin -->
-			<!-- TODO: change "pin0"..."pin11" to the values for your chip -->
           	<text x="_Middle-13" y="_Middle-100" align="CentreLeft" size="large" value="4510" />
+			<text x="_Middle-13" y="_Middle-100" align="CentreLeft" size="large" value="4510" />
```

Comment:
This is duplicated from the line above - please could you remove?

```suggestion
```

Reply:
whoops, pasted it in wrong.  commited now.